ci-operator/config/openshift/cluster-api-provider-kubevirt/openshift-cluster-api-provider-kubevirt-main.yaml

```diff
@@ -1,9 +1,82 @@
 binary_build_commands: make manager
+base_images:
+  base:
+    name: "4.10"
+    namespace: ocp
+    tag: base
+  openshift_release_golang-1.17:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
     tag: golang-1.17
+images:
+- build_args:
+  - name: ARCH
+    value: amd64
+  - name: ldflags
+  dockerfile_literal: |-
+    FROM registry.ci.openshift.org/openshift/release:golang-1.17 as toolchain
```

Comment:
I wonder if we should be following the pattern that other providers are using here?

example - https://github.com/openshift/cluster-api-provider-aws/blob/release-4.10/Dockerfile.rhel